CHANGELOG.md

```diff
@@ -6,6 +6,22 @@
 
 ### Fixes
 
+## v1.28.0 (2023-03-13)
+
+- Update TypeScript and ESLint to latest versions ([@mszabo-wikia](https://github.com/mszabo-wikia) in [#1256](https://github.com/jaegertracing/jaeger-ui/pull/1256))
```

Comment:
We should curate and categorise these into the respective Enhancements and Fixes sections below.

Reply:
I have moved them to fixes, as none of them is a user-visible improvement (as far as I understand).